x/stakeibc/types/params.go

```diff
@@ -11,7 +11,8 @@ import (
 var (
 	// these are default intervals _in epochs_ NOT in blocks
 	DefaultDepositInterval      uint64 = 5
-	DefaultDelegateInterval     uint64 = 2
+	DefaultDelegateInterval     uint64 = 5
```

Comment:
we need to raise these for robust testing; will include in my PR TEST-96

Reply:
sgtm